act-rules-format.bs

```diff
@@ -89,17 +89,40 @@ An ACT Rule MUST have a description that is in plain language and provides a bri
+
+This means that the rule maps to the accessibility requirement, as opposed to it merely being related to the requirement, thematically or otherwise. Because of this, atomic rules used in composite rules often do not map to any accessibility requirement. Failing the composite rule fails the accessibility requirement, but failing any of its atomic rules may not. In such cases the atomic rules MUST NOT list the accessibility requirements. These could be provided as background information instead.
 
-[=Atomic rules=] and [=composite rules=] SHOULD identify the accessibility requirements that fail when the outcome of a rule is Fail. An ACT Rule is a complete or partial test for one or more accessibility requirements. This means that most ACT rules test only part of an accessibility requirement, but it MUST NOT test more than the accessibility requirement it lists.
+<div class=note>
+  There are two reasons why an ACT Rule might not list any accessibility requirements:
 
-Because ACT Rules often have a smaller scope than the accessibility requirement they test, passing a rule does not necessarily mean that the accessibility requirement has passed. ACT Rules MUST indicate when they can not be used to determine that the accessibility requirement passed.
+  1. The ACT Rule is a best practice that is not required by any accessibility standard
+  2. The ACT Rule is an [=atomic rule=] and its result is used in a [=composite rule=] that does map to an accessibility requirement. See: [ACT Rule Types](#rule-types)
+</div>
 
-Outcomes from an ACT Rule SHOULD be consistent with the accessibility requirement, e.g. a rule only returns the outcome Fail when the content fails the accessibility requirement. This means that the rule maps to the accessibility requirement, as opposed to it merely being related to the requirement, thematically or otherwise. Because of this, atomic rules used in composite rules often do not map to any accessibility requirement. Failing the composite rule fails the accessibility requirement, but failing any of its atomic rules may not. In such cases the atomic rules MUST NOT list the accessibility requirements. These could be provided as background information instead.
+<div class=note>
+  In WCAG 2, success criteria do not evaluate to Pass, Fail or Inapplicable. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) A Fail result from an ACT Rule means the success criteria listed as "accessibility requirements" were not satisfied for the particular page. A page may still conform to WCAG 2 if a conforming alternative version is provided, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
```

Comment:
Should there also be a note about "Inapplicable" mapping to "satisfied" in WCAG 2?

Reply:
In [rule aggregation](https://pr-preview.s3.amazonaws.com/w3c/wcag-act/pull/276.html#rule-aggregation) we write "Example: An expert evaluates a success criterion to fail on a specific page." This needs to be updated to match this new definition.